Gemfile

```diff
@@ -34,4 +38,5 @@ gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 # do not have a Java counterpart.
 gem "http_parser.rb", "~> 0.6.0", :platforms => [:jruby]
 
-gem "webrick", "~> 1.7"
+# Add webrick to fix a bug with Ruby version 3.0.0 or higher
+gem "webrick", "~> 1.7"
```

Comment:
Could you add a blank line at the end of this file? 🙏

Reply:
Yes, will do. Thanks @dirtyhenry. 👍